src/agents/ipp_group.jl

```diff
@@ -5110,13 +931,14 @@ function solve_agent_problem_ipp_cap(
             ipp.eximport_my(y, z, d, t) +
             # total DG generation at time t
             sum(
-                customers.rho_DG(h, m, z, d, t) * customers.total_der_capacity_my(y, z, h, m) for
+                customers.rho_DG(h, m, z, d, t) * customers.total_der_capacity_my(cust_year, z, h, m) for
```

Comment:
I would recommend porting my changes over. There're many tricky things to consider when aligning the lower level LP with previous bi-level problem. For example, when the bi-level problem is solved for year `y`, `customers.total_der_capacity_my` is the sum of existing DER capacity from year `y`, and new DER build from year `y-1` (because IPP is solved before Customer). However, if we're running the lower level LP at year `y+1`, even with pointing `customers.total_der_capacity_my` to year `y`, that value will now incorporate new DER build from year `y` as well (because Customers' problem have solved), causing discrepancies between the lower level LP and previous year's bi-level problem. In my update, I looked into the parameters in many JuMP constraints to make sure those are aligned, and therefore lower level LPs should always return feasible solution if previous bi-level problem is optimal.